configuration/jsr223.md

```diff
@@ -402,6 +402,17 @@ sharedCache.remove("x")
 
 :::
 
+::: tab DSL
+
+```java
+sharedCache.put('foo', 'bar')
+sharedCache.get('foo') // returns null if doesn't exist
+shareCache.put('foo', null) // deletes the entry
+```
+
+:::
+
+
```

Comment:
Can you please remove that line, Iine? Otherwise it looks good to me. Thanks for adding it.

Reply:
Never mind, @udo1toni, I was able to do it myself.